src/main/resources/db/changelog/changes/10-create-shopping-carts-table.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: Alex
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                    foreignKeyName: fk_users_id
+                    references: users(id)
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValue: false
+                  constraints:
+                    nullable: false
```

Comment:
```suggestion
                    nullable: false

```
![image](https://github.com/user-attachments/assets/819573d3-d6f9-4957-ba6b-a445779952e7)
if you see this red symbol in PR, it means you need to add one more line